src/r-rig/install.sh

```diff
@@ -45,6 +46,16 @@ if [ "${R_VERSION}" = "latest" ]; then
 fi
 
 # Check options for installing packages
+if [ "${VSCODE_R_SUPPORT}" = "minimal" ]; then
+    R_PACKAGES+=(jsonlite rlang)
+elif [ "${VSCODE_R_SUPPORT}" = "languageserver" ]; then
+    R_PACKAGES+=(languageserver)
+    APT_PACKAGES+=(libxml2-dev libicu-dev)
+elif [ "${VSCODE_R_SUPPORT}" = "full" ]; then
```

Comment:
"full" seems like a reasonable choice to me.  Naming things is hard and one day we may wish we had a more granular set of options, but I think we should go with this.

Reply:
My concern is that the source installation of `languageserver` and `httpgd` takes a very long time, but if I try to install `languageserver` after the container is built, it fails because `libxml2-dev` is not installed, so it would certainly make more sense to have it installed from the beginning than to install it later.